src/components/tutorial/FeedbackButton.astro

```diff
@@ -377,10 +406,16 @@ const instanceId = Math.round(Math.random() * 10e9).toString(36);
 
 	.select-buttons > a:hover,
 	.select-buttons > button:hover {
+		text-decoration: none;
 		background-color: var(--sl-color-gray-5);
 	}
 
 	.select-buttons h2 {
+		font-size: var(--sl-text-h5);
+		font-weight: 600;
+		line-height: var(--sl-line-height-headings);
+		margin-bottom: 0.5rem;
+		text-decoration: none;
```

Comment:
I’m not quite clear what the changes _inside_ the feedback modal are doing? Comparing the preview and the live site, the live site’s styles seem correct, so I think we should aim to match them (which is maybe as simple as not changing anything?)

Reply:
I discovered that moving the `FeedbackButton` outside the RightSideBar made some scoped CSS in-accessible. This made the formatting for the title in correct. The above fixes this. It is the same style as before (take directly from inspect). I'm not sure if there is some other way to keep the formatting.